_includes/head.html

```diff
@@ -11,7 +11,7 @@
 
   <link rel="icon" type="image/png" href="data:image/png;base64,iVBORw0KGgo=">
 
-  <link rel="stylesheet" href="/styles.css">
+  <link rel="stylesheet" href="{{ site.baseurl }}/styles.css">
```

Comment:
When looking at the source of the [deploy preview](view-source:https://deploy-preview-14--digital-garden-jekyll-template.netlify.app/), it appears that the generated line looks like this:

```
<link rel="stylesheet" href="//styles.css">
```

This `href` property points to a 404, and the page doesn't have any styling. To fix this, we should remove the `/` to make sure we only use a single slash (the one from the `baseurl`):

```suggestion
  <link rel="stylesheet" href="{{ site.baseurl }}styles.css">
```